addons/jest/src/components/Panel.tsx

```diff
@@ -107,64 +94,77 @@ interface ContentProps {
-              (successNumber / result.assertionResults.length) *
-              100
-            ).toFixed(2)}%`}</PassingRate>
-            <SuiteProgress {...{ successNumber, failedNumber, result }} />
-            <SuiteTotals {...{ successNumber, failedNumber, result }} />
-          </SuiteHead>
-          <List>
-            {result.assertionResults.map(res => (
-              <Item key={res.fullName || res.title}>
-                {res.failureMessages && res.failureMessages.length ? (
-                  <FailedResult {...res} />
-                ) : (
-                  <Result {...res} />
-                )}
-              </Item>
-            ))}
-          </List>
-        </section>
+        <SizeMe refreshMode="debounce">
+          {({ size }: { size: any }) => {
+            const { width } = size;
+            return (
+              <section key={name}>
+                <SuiteHead>
+                  <SuiteTotals {...{ successNumber, failedNumber, result, passingRate, width }} />
+                  {width > 240 ? (
+                    <ProgressWrapper>
+                      <SuiteProgress {...{ successNumber, failedNumber, result }} />
+                    </ProgressWrapper>
+                  ) : null}
+                </SuiteHead>
+                <TabsState initial="failing-tests" backgroundColor="rgba(0,0,0,.05)">
+                  <div id="failing-tests" title={`${failedNumber} Failed`} color="#FF4400">
+                    <List>
+                      {result.assertionResults.map(res => {
+                        return res.status === 'failed' ? (
+                          <Item key={res.fullName || res.title}>
+                            <Result {...res} />
+                          </Item>
+                        ) : null;
+                      })}
+                    </List>
+                  </div>
+                  <div id="passing-tests" title={`${successNumber} Passed`} color="#66BF3C">
+                    <List>
+                      {result.assertionResults.map(res => {
+                        return res.status === 'passed' ? (
+                          <Item key={res.fullName || res.title}>
+                            <Result {...res} />
+                          </Item>
+                        ) : null;
+                      })}
+                    </List>
+                  </div>
+                </TabsState>
+              </section>
+            );
+          }}
+        </SizeMe>
       );
     })}
   </div>
 ))({
-  padding: '10px 20px',
   flex: '1 1 0%',
 });
 
+const ContentWithTheme = withTheme(Content);
+
 interface PanelProps {
   tests: null | Test[];
 }
 
 const Panel = ({ tests }: PanelProps) => (
   <ScrollArea vertical>
-    {tests ? <Content tests={tests} /> : <NoTests>This story has no tests configured</NoTests>}
+    {tests ? (
+      <ContentWithTheme tests={tests} />
+    ) : (
+      <NoTests>This story has no tests configured</NoTests>
```

Comment:
It would be nice to add some documentation here, on how to get it set up so there ARE tests.